src/shapes/IText/ITextBehavior.ts

```diff
@@ -313,19 +313,23 @@ export abstract class ITextBehavior<
    * @param {Number} direction 1 or -1
    * @return {Number} Index of the beginning or end of a word
    */
-  searchWordBoundary(selectionStart: number, direction: number): number {
+  searchWordBoundary(selectionStart: number, direction: 1 | -1): number {
```

Comment:
we may want to safeguard `direction = Math.sign(direction )`

Reply:
maybe no, here is type there is doc, you are welcome to break your own software if you really don't want to know how this is supposed to work.